src/conventional/sonde_tac2ioda.py

```diff
@@ -723,11 +728,16 @@ def getDayHour(str):
     Parse day of month and hour from the given string
     :param str: The string to parse
     :return: (day, hour)
+    When the day is coded as day+50, windspeed is in knots, otherwise m/s.
     """
+    global scale_wspd
     try:
         day = int(str[:2]) - 50
         if day < 0:
             day += 50
+            scale_wspd = 1.0
+        else:
+            scale_wspd = knots2mps
```

Comment:
Just trying to understand this: so if the day is negative, the unit is mps; vice versa. Is it correct?

Reply:
Ok. I understand now. They use "50" as a trick to distinguish the use of the different units. Thanks!